shoov/modules/custom/shoov_ci_build/shoov_ci_build.module

```diff
@@ -25,6 +25,22 @@ function shoov_ci_build_menu() {
     'type' => MENU_CALLBACK,
   );
 
+  $items["ci_build/%ci_build/%uid/subscribe/%ctools_js"] = array(
```

Comment:
this requires also a unique token, to prevent sending links to the admin, where they will click and un/subscribe the user. The unique token (e.g. md5(uid + ci_build)) will prevent that.